spmd/tensor/parallel/api.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
+import torch
+import torch.nn as nn
+import functools
+from typing import Sequence, Tuple, Callable
+from spmd import (
```

Comment:
nit: let's tensor.parallel should ideally only depends on `tensor` subpackage, let's directly import from `spmd.tensor` instead.